sensu-plugins-cpu-checks.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsCpuChecks::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin', ['>= 1.2', '< 2.0']
```

Comment:
good call, I somehow read it has `'~> 1.2.0'` which is not the same as I proposed.